Library/Homebrew/formula_auditor.rb

```diff
@@ -900,6 +902,81 @@ def audit_revision
             current_revision > (newest_committed[:revision] + 1)
         problem "`revision` should only increment by 1"
       end
+
+      revision_increment = if (previous_revision = previous_committed[:revision])
```

Comment:
I think this should be:

```suggestion
      revision_increment = if (previous_revision = newest_committed[:revision])
```

Say we have the following commit log (most recent on top):

- `c` (PR branch): `1.0.1`
- `b` (`origin/HEAD`): `1.0.1`
- `a`: `1.0.0_1`

`previous_committed[:revision]` will be `1`, but `newest_committed[:revision]` will be `0`.

We don't need to continue on this check in this case, because whatever change happened on commit `c` didn't touch `revision`